test_data/analysis_scene4_competitive.py

```
@@ -0,0 +1,190 @@
+# Jan vs Jun comparison
+print("\nJan → Jun shift:")
+for brand in run_dem.index:
+    jan = run_dem.loc[brand, 1] if 1 in run_dem.columns else None
+    jun = run_dem.loc[brand, 6] if 6 in run_dem.columns else None
+    if jan and jun:
+        arrow = "↑" if jun > jan else "↓"
+        print(f"  {brand:<14} {jan:.2f} → {jun:.2f}  {arrow}")
+
+# Brands over-indexed with 18-24 (index > 1.0) in Running
+over_indexed = running.groupby('brand')['demographic_index_1824'].mean()
+print(f"\nOver-indexed with 18-24 (avg index > 1.0):")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)